src/utils.jl

```diff
@@ -21,6 +21,22 @@ function unique_indices(x)
     return unique, indices
 end
 
+"""
+    split_chains(data::AbstractArray{<:Any,3}, split::Int=2)
+
+Split each chain in `data` of shape `(ndraws, nchains, nparams)` into `split` chains.
+
+If `ndraws` is not divisible by `split`, the last `mod(ndraws, split)` iterations are
```

Comment:
In particular if the fumction is not exported, it would be good to explain these things in the documentation of `ess_rhat`.